drivers/hyperv/hyperv.go

```diff
@@ -560,3 +558,50 @@ func (d *Driver) generateDiskImage() (string, error) {
 
 	return diskImage, nil
 }
+
+func writeSSHKeyToVHDX(vhdxPath, publicSSHKeyPath string) error {
+	mountDir := "E:\\"
```

Comment:
This will fail if I have an existing drive E on my computer. Can we retrieve the assigned drive letter after mount?